backend/src/test/scala/cromwell/backend/MemorySizeSpec.scala

```diff
@@ -65,26 +53,32 @@ class MemorySizeSpec extends FlatSpec with Matchers with TryValues {
   it should "convert to other units properly" in {
     val memTable = Table(
       ("memorySize", "newUnit", "result"),
-      (MemorySize(1000, MemoryUnit.Bytes), MemoryUnit.KB, MemorySize(1, MemoryUnit.KB)),
-      (MemorySize(1000000, MemoryUnit.Bytes), MemoryUnit.MB, MemorySize(1, MemoryUnit.MB)),
-      (MemorySize(1024, MemoryUnit.Bytes), MemoryUnit.KiB, MemorySize(1, MemoryUnit.KiB)),
-      (MemorySize(1, MemoryUnit.KB), MemoryUnit.Bytes, MemorySize(1000, MemoryUnit.Bytes)),
-      (MemorySize(1, MemoryUnit.MB), MemoryUnit.Bytes, MemorySize(1000000, MemoryUnit.Bytes)),
-      (MemorySize(1, MemoryUnit.KiB), MemoryUnit.Bytes, MemorySize(1024, MemoryUnit.Bytes))
+      (MemorySize(1024, MemoryUnit.Bytes), MemoryUnit.KB, MemorySize(1, MemoryUnit.KB)),
+      (MemorySize(1048576, MemoryUnit.Bytes), MemoryUnit.MB, MemorySize(1, MemoryUnit.MB)),
+      (MemorySize(1, MemoryUnit.KB), MemoryUnit.Bytes, MemorySize(1024, MemoryUnit.Bytes)),
+      (MemorySize(1, MemoryUnit.MB), MemoryUnit.Bytes, MemorySize(1048576, MemoryUnit.Bytes))
     )
 
     forAll(memTable) { (memorySize, newUnit, result) =>
       memorySize.to(newUnit) shouldEqual result
     }
   }
+  
+  it should "round trip" in {
+    List(
+      "2 GB"
```

Comment:
Seems like this is a test setup which might be expecting more than one test case?